draft-ietf-taps-interface.md

```diff
@@ -2332,27 +2332,59 @@ a data transfer useful. It is given in bits per second. The special value -1 ind
 that no bound is specified.
 
 
-### TCP-specific Property: User Timeout {#tcp-uto}
+## TCP-specific Properties: User Timeout Option (UTO) {#tcp-uto}
 
-This property specifies, for the case TCP becomes the chosen transport protocol:
+These properties specifies configurations for the User Timeout Option (UTO), 
+in case TCP becomes the chosen transport protocol as an example for a protocol-specific property. 
```

Comment:
It's not an example, it's one out of many properties in the list, and we designated this as being TCP-specific. I would remove everything from (and including) "as an example" in this sentence.

Reply:
Removed this again but I think we should say something about why this is the one and only TCP property that is included.